single-function/add-two-numbers/02-impl-revision.js

```diff
@@ -1,24 +1,24 @@
 const addTwoNumbers = (l1, l2) => {
-  let carryover = 0,
-    dummyhead = { val: 0, next: null },
-    curr = dummyhead;
+  let carryOver = 0;
+  const dummyHead = { val: 0, next: null };
+  let curr = dummyHead;
 
   while (l1 !== null || l2 !== null) {
-    let list1Value = l1 !== null ? l1.val : 0;
-    let list2Value = l2 !== null ? l2.val : 0;
-    let sum = list1Value + list2Value + carryover;
-    carryover = Math.floor(sum / 10);
+    const list1Value = l1 !== null ? l1.val : 0;
```

Comment:
More idiomatic to just say 
If using "val" elsewhere, for consistency let's just stick with "val" here too.  Or make it "value" everywhere. 

```
const list1Val = list1 ? list1.val : 0;
```

Reply:
yeah val is super common.